docs/dab-cli.md

```diff
@@ -96,6 +99,20 @@ Update the properties of any database entity in the configuration file.
 | **--relationship.fields** | false   | -   | Specify fields to be used for mapping the entities. Example: `--relationship.fields "id:book_id"`. Here `id` represents column from sourceEntity, while `book_id` from targetEntity. Foreign keys are required between the underlying sources if not specified.  |
 | **-m, --map** | false   | -   | Specify mappings between database fields and GraphQL and REST fields. format: --map "backendName1:exposedName1,backendName2:exposedName2,...".   |
 
+### **`export`**
+Export the GraphQL schema as a file and save to disk.
```

Comment:
By default, if we dont specify the `--graphql` option, this command does not do anything. 

`dab export -o .\schemas` => is a no-op and yet is a valid command.

So, the description for this command should not be limited to just graphql -> `export the required schema as a file and save to disk based on the options`. 

In future, this can be used to export the OpenAPI Spec documentation.

Reply:
updated.